app/src/main/java/com/example/myfirstapp/ShoppingTabFragment.java

```diff
@@ -2,16 +2,121 @@
 
 import android.os.Bundle;
 import android.support.v4.app.Fragment;
+import android.support.v4.content.ContextCompat;
+import android.support.v7.widget.DefaultItemAnimator;
+import android.support.v7.widget.LinearLayoutManager;
+import android.support.v7.widget.RecyclerView;
+import android.text.Editable;
+import android.text.InputType;
+import android.text.TextWatcher;
 import android.view.LayoutInflater;
 import android.view.View;
 import android.view.ViewGroup;
-
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.TextView;
 
 /**
  */
 public class ShoppingTabFragment extends Fragment {
+
+    // Adapter
+    SimpleAdapter recyclerAdapter;
+
+    // RecyclerView
+    RecyclerView recyclerView;
+
+    // Field for entering item
+    EditText editShoppingItemName;
+
+    // Field for entering price of item
+    EditText editShoppingItemPrice;
+
+    // Title of the list area
+    TextView listTitle;
+
+    // Button for adding item
+    Button addItem;
+
     @Override
     public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle bundle) {
-        return inflater.inflate(R.layout.fragment_shopping_tab, container, false);
+        super.onCreate(bundle);
+        final View view = inflater.inflate(R.layout.fragment_shopping_tab, container, false);
+
+        editShoppingItemName = view.findViewById(R.id.editItemName);
+        editShoppingItemPrice = view.findViewById(R.id.editText);
+        listTitle = view.findViewById(R.id.list_title);
+        addItem = view.findViewById(R.id.add_button);
+
+        // Change fields based on view
+        editShoppingItemPrice.setHint(R.string.edit_text_add_price);
+        editShoppingItemPrice.setInputType(InputType.TYPE_CLASS_NUMBER | InputType.TYPE_NUMBER_FLAG_DECIMAL);
+
+        editShoppingItemPrice.addTextChangedListener(new TextWatcher() {
+            @Override
+            public void beforeTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+
+            }
+
+            @Override
+            public void onTextChanged(CharSequence charSequence, int i, int i1, int i2) {
+
+            }
+
+            @Override
+            public void afterTextChanged(Editable editable) {
+                editShoppingItemPrice.removeTextChangedListener(this);
+                editShoppingItemPrice.setText("$ " + editShoppingItemPrice.getText().toString());
```

Comment:
You can make that `$` a string res and then you can use `String.format()`

Your string res might look something like:
`<string name="money_prefix">$%s</string>`
and the String.format would look something like this:
`String.format(Locale.US, getString(R.string.money_prefix), editShoppingItemPrice.getText().toString())`